### ISSUE TITLE
lb-ng never exits v2.2.0

### ISSUE THREAD

**Issue by rhansma** [strongloop/loopback-sdk-angular-cli#59]
Hi,

It seems that version 2.2.0 reintroduces a bug which was fixed earlier. This bug prevents the lb-ng script from exiting. Instead it will start the server after generating the lbServices file.

I've investigated the issue myself and I found that replacing the code on line 70-75 (https://github.com/strongloop/loopback-sdk-angular-cli/blob/master/bin/lb-ng.js#L70) with the code of v2.0.1, lines 55-57 (https://github.com/strongloop/loopback-sdk-angular-cli/blob/v2.0.1/bin/lb-ng#L55) does restore the expected behaviour.

**Comment by KuroNoDev** [comment]
This is a huge help to me. Thank you.

**Comment by rhansma** [comment]
Happy to help!

**Comment by davidcheung** [comment]
@rhansma the change was introduced to fix an issue with node 6 getting exited from nextTick before stdout is done when script content is too long, looks like we need to seek an alternative if the code is not exiting. 

Would you be able to provide steps to reproduce this issue?

**Comment by rhansma** [comment]
@davidcheung Ah ok, already thought that change was made deliberately. I will describe the steps to reproduce below.

**NodeJS version:** 4.6.1
**loopback-sdk-angular-cli version:** 2.2.0
When a non exiting boot script is used, the lb-ng command hangs. For example when the code below is placed in a boot script, the lb-ng command wont exit when using version 2.2.0. When using version 2.0.1 it exists correctly after generating the lbServices.js file.
```
module.exports = function(server) {
  setInterval(function() {
  	console.log('test');
  }, 1000);
};
```
I've indeed found that when using NodeJS v6.9.1 the script correctly exists.

**Comment by dtomasi** [comment]
I also have this problem.`lb-ng` is not exiting in CLI.
My environment is: 

- macOS Sierra 10.12.1
- NodeJS 6.9.1 via homebrew

```
slc --version
strongloop v6.0.3 (node v6.9.1)
├── strong-arc@1.8.9
├── strong-build@2.1.2
├── strong-deploy@3.1.4
├── strong-mesh-models@8.1.0
├── strong-pm@5.2.3
├── strong-registry@1.1.6
├── strong-start@1.3.4
├─┬ strong-supervisor@3.3.3
│ └── strong-agent@2.1.0
├── generator-loopback@1.24.4
├── node-inspector@0.7.4
└── nodefly-register@0.3.3
```

My current workaround is using `coreutils` homebrew package.

`brew install coreutils`

`gtimeout 5 lb-ng --url https://api.example.com/api server/server.js lb-services.js`

**Comment by davidcheung** [comment]
hmm maybe we should revert the change, but then node 6 will generate incomplete sdks due to this 

cc @bajtos

**Comment by bajtos** [comment]
Considering how much time we have already spent trying to figure out how to correctly support the feature of printing the generated script to `stdout`, and the little value it brings to users (is there anybody actually using it?), I am proposing to remove this feature completely and release a new major version of loopback-sdk-angular-cli.

@davidcheung thoughts?

**Comment by davidcheung** [comment]
I agree @bajtos, yeah perhaps we should revert the feature and revisit at a different time, having it not exit is pretty bad.

**Comment by bajtos** [comment]
This should have been fixed by #62, please reopen if the problem persists.
